### PR TITLE
Update README following scrum master meeting notes

### PR DESCRIPTION
**Pull request by Cleop**
- Creates contents, #25 
- Amends priority labels to reflect dwyl's usage of priority-1, #31 
- Details time spent to be a SM, #30 
- Adds notes to section on stand-ups, #26 
- Adds notes to retrospective section, #29 
- Adds list of responsibilities, #4 #28 
- Adds notes to sprint planning, #32 
- Does a once over edit to make sure everything makes sense, #34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,43 +4,107 @@ Contains our processes, questions and journey to creating ateam.
...
+
+- **Total understanding of the problem (that the product is trying to solve)**
+- **Lifting blockers**
+   - Ie. making sure the team has everything they need to progress with their work, no outstanding external dependencies, all the designs, copy, login details, card details they need and decisions made on issues that are in discussion.
+- **Enabling the team to focus on coding rather than project management**
+   - Ie. the SM should be responsible for stopping scope creep or liaising with the client over wider project issues so that the team can focus on executing issues that they know are ready to go.
+- **Enforcing good gitflow across both teams**
+   - Ie. making sure the client is using github as dwyl sets out and that the team are too (correct labelling, good commit messages etc.)
+- **Full knowledge of the product backlog to enable them to efficiently do backlog grooming, sprint planning and time estimating.**
+- **Backlog grooming**
```
Comment (iteles) [inline README.md]: I think it's worth highlighting the asking questions early on in the sprint as one of the top things here - you raised an excellent point when we were talking @cleop that developers might not get to some lower priority issues until late in the sprint and all the info needs to be ready to go

File: README.md [hunk 2]
```diff
@@ -4,43 +4,107 @@ Contains our processes, questions and journey to creating ateam.
...
+   - Ie. making sure the backlog doesn't have duplicates, issues are correctly labelled (prioritised, time estimated etc) and have clear and full descriptions with wireframes, designs or assets where appropriate, that new issues are created rather than tagged on to old issues, asking any outstanding questions.
+- **Setting up project meetings**
+   - Ie. when/ where sprint demos/planning or other client meetings, stand-ups take place.
+- **Agrees sprint scope during sprint planning**
+- **Client communication, on hand to answer questions that may arise**
+
+A SM does not have 'authority' over the team, they work collaboratively with them. When the team needs direction to keep on track with the sprint milestone or with good gitflow practices they should be guided not forced! 😊
+
+## Ceremonies
+**Ceremonies** are the events that scrum masters take part in every sprint.
```
Comment (iteles) [inline README.md]: They are more like meetings that the _team_ take part in during each sprint.

File: README.md [hunk 3]
```diff
@@ -4,43 +4,107 @@ Contains our processes, questions and journey to creating ateam.
...
+
+## Stand-ups
+
+- Scheduling stand ups is the SM's responsibility
+- To be held daily at the beginning or end of the day
+- Should not last longer than 15mins
```
Comment (iteles) [inline README.md]: More like 3 minutes per team member, you can get some pretty big teams ;) Or some tiny ones!

File: README.md [hunk 4]
```diff
@@ -4,43 +4,107 @@ Contains our processes, questions and journey to creating ateam.
...
+- Can be done remotely via google hangouts, for which screensharing is recommended so people can see what everyone's been working on
```
Comment (iteles) [inline README.md]: via a remote conferencing tool, such as g hangouts 😉

File: README.md [hunk 5]
```diff
@@ -4,43 +4,107 @@ Contains our processes, questions and journey to creating ateam.
...
+
+**Each team member briefly summarises:**
+  - The issues they've been working on that day
+  - What issues they intend to work on the following day
+  - Whether they experienced any blockers or have any questions
+
+**The SM should:**
+  - Update the team on their progress lifting any blockers e.g. progress getting an outstanding external dependency from the client.
+  - Ask questions as the team show the work they've done that day where relevant (ie. have you written documentation for that issue? did you update the readme to reflect those changes?)
```
Comment (iteles) [inline README.md]: I think we can think of some question examples here that are less 'Did you follow process?' and more probing. Things like 'Does that change therefore impact the timelines/effort for this other feature in the sprint?' or 'What is the result if the user chooses _not_ to fill in all the fields of that form?'

Comment (iteles) [inline README.md]: Note: the  readme comment should be left in for sure as it's a good reminder that it often needs to be asked, just not as the main thing a SM asks. I only ask about it after having quickly checked the PR in question if I can, but mostly for the more complex features.

File: README.md [hunk 6]
```diff
@@ -4,43 +4,107 @@ Contains our processes, questions and journey to creating ateam.
...
++ **Decide who will be driving the demo** :blue_car:
 The demo will usually be introduced by the scrum master or the product owner (_the person with the most demo experience on the team_). The demo will then be handed over to the developers who have been working on it as they begin to go through the user flow. At any given time, the person with the most experience on a certain aspect of the application should be talking about it.
 
   We recommend that the person who is talking is a different person to the one driving (performing the actions in the application). When using the application it can be easy to stop interacting with the people you are demo-ing to and speed up the pace of the demo as you click through an application you personally know well. This will need to be prepared and practiced in advance.
-+ **Do a run-through of the demo**  
-Once you've decided who is driving/ speaking you should do a run-through. This is where you'll pick up all of your problems around flow and most importantly, unexpected bugs that you don't want showing up in the actual demo.
-+ **Recap on lessons learnt from previous demos**  
++ **Do a run-through of the demo** :loop:
```
Comment (iteles) [inline README.md]: Loving the addition of all the emojis 😆

File: README.md [hunk 7]
```diff
@@ -86,17 +150,37 @@ Once the previous sprint's stories have been reviewed and moved to the relevant
...
-+ [x] Who drives? Definitely you!
-+ [x] How to wrap up
-+ [ ] Release notes
+As soon as the Product Owner (client) is satisfied that the agreed stories for the sprint have been delivered, prepare and send the ***invoice before anything else***. NB. This may not fall as the responsibility of the SM.
+
+## Retrospective
+
+- A retrospective is held after the sprint demo
+- Can be held either all-in-one with the client or in two parts, one with the client, one without depending on the client/team's preferences.
+- They should last between **7 - 30 mins**.
```
Comment (iteles) [inline README.md]: 7 was a bit of an arbitrary joke 😊 
Let's stick with 10 - 30 please!

File: README.md [hunk 8]
```diff
@@ -86,17 +150,37 @@ Once the previous sprint's stories have been reviewed and moved to the relevant
...
+The duration varies according to the size of the team and often may be longer earlier on in a project when the team is settling in.
+- During the retrospective each team member should have the opportunity to share things that they felt **went well**, **not very well** or things that could be **improved** based on the last sprint.
+
+## Sprint Planning
+Spring planning is when you review the backlog and client's priorities to outline what will be put into the next sprint milestone. This may be done immediately after a sprint demo and retrospective or otherwise on a time/date agreed at the end of the sprint demo.
+
+### <a name="sprint-planning-prep"></a>Sprint Planning Prep
```
Comment (iteles) [inline README.md]: I don't think we need this anchor tag, as it's a title, we can just use the title as the anchor #sprint-planning-prep (supported by gh markdown at least until recently)

File: README.md [hunk 9]
```diff
@@ -86,17 +150,37 @@ Once the previous sprint's stories have been reviewed and moved to the relevant
...
+- Use the opportunity to ask the client any questions that the team would need to know in order to implement the various issues.
+- Ensure that issues are clearly defined so that time estimates reflect the implementation of the issue clearly. Ie. add a checkbox to user sign up so people can subscribe to the newsletter. Does this mean just add the checkbox or does this mean implement an admin dashboard to manage newsletter signs up and enable users to sign up by adding a checkbox on sign up?
+- Try to bring back product conversations that may not be relevant or may not require everyone present at the meeting to help keep to time.
+- Make note of the issues and time estimates for your reference (the Product Owner should create/update the issues on github but you may need the notes before they've had a chance to do this).
+- Record any bugs/ enhancements that were discussed during the sprint demo to suggest as new issues during sprint planning. If it takes your Product Owner time to update github after your sprint planning you can set the team to work on these small issues that should be relatively straightforward to complete.
```
Comment (iteles) [inline README.md]: Slight objection to the wording here "you can set the team to work" - that's not quite it, more like "the team can start working on"... remember, zero authority over the team, full authority over the process

File: README.md (hunk 3 above)
Comment (nelsonic) [inline README.md]: 3 minutes is an _eternity_ in standup time! I've never worked on a team where it was _acceptable_ to take longer than a _minute_ per person.

Comment (iteles) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (Cleop) [inline README.md]: Made this change in a line or two above 👍  - but won't remove this comment as the comment is attached to this line.

**Comment by Cleop** [comment]
Ready for you to check out again @iteles 😊

File: README.md [hunk 10]
```diff
@@ -35,7 +35,7 @@ So what is a scrum master responsible for?
 
 - **Total understanding of the problem (that the product is trying to solve)**
 - **Lifting blockers**
-   - Ie. making sure the team has everything they need to progress with their work, no outstanding external dependencies, all the designs, copy, login details, card details they need and decisions made on issues that are in discussion.
+   - Ie. making sure the team has everything they need to progress with their work, no outstanding external dependencies, all the designs, copy, login details, card details they need and decisions made on issues that are in discussion. Sometimes developers may only come to lower priority issues towards the end of a sprint. You need to ensure that those issues are ready to be tackled with all the required information gathered at the beginning of the sprint. This ensures that the developers don't have to waste valuable time chasing or clarifying things when they come to work on them.
```
Comment (nelsonic) [inline README.md]: @Cleop please see: https://github.com/dwyl/style-guide/issues/9
(_one sentence per line_)

**Comment by Cleop** [comment]
Thanks for the spot @nelsonic, I've given it a little edit to try and make it a bit more readable 😊

**Pull request merged**